terraform/terraform.tfvars

```diff
@@ -1,8 +1,9 @@
 codebuild_cache_bucket_name="apprunner-cache-yyyymmdd-identifier"
-aws_region="us-east-1"
+aws_region="us-east-2"
```

Comment:
Do we want to move to us-east-2? This will need to align with how we provision accounts in Event Engine, as well as references throughout the readme, particularly in the prerequisites section.

Reply:
That's a variable, which users can change based on their preference and app runner availability

Reply:
i can change it to us-east-1